tests/drum/test_unstructured_mode.py

```diff
@@ -44,7 +44,7 @@ def test_unstructured_models_batch(
 
         output = tmp_path / "output"
 
-        cmd = "{} score --code-dir {} --input {} --output {} --unstructured".format(
+        cmd = "{} score --code-dir {} --input {} --output {} --target-type unstructured".format(
```

Comment:
Not specific for this line, but there are not tests for the case we need to guess the target type and fail + no other tests with --target-type?

Reply:
I'll review it deeper, as looks like it is better to plumb `target_type` trough all the predictors, by doing:
```
if binary:
   predict binary
elif regression:
   predict regression
```

Reply:
did it